frontend/packages/console-shared/src/components/pod/PodStatus.tsx

```diff
@@ -120,7 +120,8 @@ class PodStatus extends React.Component<PodStatusProps, PodStatusState> {
         ]
       : undefined;
     return (
-      <ChartDonut
+      <OptimizedChartDonut
```

Comment:
This was created instead of PureComponent'ing this component as an speedy way to compare just before going into the rendering of the Patternfly chart (Victory chart)

Reply:
Main flaw with that I did was the blocking of some props during update. Christian suggested a solution that panned out - updated PR.